test-requests.js fails when run alone locally

● should handle batchPlugin function error
            Chrome 67.0.3396 (Mac OS X 10.13.4)
          AssertionError: expected spy to have been called exactly once, but it was called 0 times

--- a/extensions/amp-analytics/0.1/test/test-requests.js
+++ b/extensions/amp-analytics/0.1/test/test-requests.js
@@ -369,6 +369,7 @@ describes.realWin('Requests', {amp: 1}, env => {
             analyticsMock, r, preconnect, spy, false);
         // Overwrite batchPlugin function
         handler.batchingPlugin_ = () => {throw new Error('test');};
+        expectAsyncConsoleError(/test/);
         const expansionOptions = new ExpansionOptions({});
         handler.send({}, {'extraUrlParams': {'e1': 'e1'}}, expansionOptions);
         clock.tick(1000);

